libparsec/src/config.rs

```diff
@@ -31,3 +31,53 @@ impl From<ClientConfig> for libparsec_client::ClientConfig {
         }
     }
 }
+
+pub fn get_default_data_base_dir() -> PathBuf {
+    #[cfg(target_arch = "wasm32")]
+    {
+        PathBuf::from("")
+    }
+    #[cfg(not(target_arch = "wasm32"))]
+    {
+        // TODO: Windows path differs from Parsec v2 here:
+        // - previous: `%APPDATA%\parsec\data`
+        // - current: `%APPDATA%\Roaming\parsec-v3-alpha`
+        // TODO: check MacOS path correspond to Parsec v2
+        let mut path = dirs::data_dir().expect("Could not determine base data directory");
+        // TODO: temporary name to avoid clashing with stable parsec
+        path.push("parsec-v3-alpha");
+        path
+    }
+}
+
+pub fn get_default_config_dir() -> PathBuf {
+    #[cfg(target_arch = "wasm32")]
+    {
+        PathBuf::from("")
+    }
+    #[cfg(not(target_arch = "wasm32"))]
+    {
+        // TODO: Windows path differs from Parsec v2 here:
+        // - previous: `%APPDATA%\parsec\config`
+        // - current: `%APPDATA%\Roaming\parsec-v3-alpha`
```

Comment:
`%APPDATA%`, contrary to what the name suggests, points to `C:\Users\<user>\APPDATA\Roaming`

Reply:
```suggestion
        // - current: `%APPDATA%\parsec-v3-alpha`
```